c7n/resources/ebs.py

```diff
@@ -454,6 +454,86 @@ def process(self, resources, event=None):
         return self.get_matching_aliases(resources)
 
 
+@Snapshot.filter_registry.register('invalid-kms')
+class KmsKeyActiveSnapshot(FilterKmsInvalid):
+    """
+    Filter to ignore snapshots that have an active KMS Key.
+
+    This filter is 'true' by default.
+
+    :example:
+
+    implicit with no parameters, 'true' by default
+
+    .. code-block:: yaml
+
+            policies:
+              - name: delete-snapshots-with-missing-keys
+                resource: ebs-snapshot
+                filters:
```

Comment:
you can move the doc string and schema to the base class, and just register the base class directly, there really isn't any need for an empty subclass.